flybywiresim/aircraft
# feat(issueTemplates): Update bug report templates to include simulator versions

## Summary of Changes

- Added the prompt in to both bug_report_a32nx.yaml and bug_report_a380x.yaml

## Screenshots (if necessary)
![image](https://github.com/user-attachments/assets/0df7e3a6-f9c8-46e1-b374-bdba7595445a)
![image](https://github.com/user-attachments/assets/50635fd1-a0d4-4efd-901c-df688654e3b8)

## References
Not required

## Additional context

Discord username (if different from GitHub): captcrystal

## Testing instructions
I don't believe QA would be required. If it is then follow these instructions:

- Open up a test bug report for the A32NX
- Confirm both options are "clickable" and that they do their intended function
- Open up a test bug report for the A380X
- Confirm both options are "clickable" and that they do their intended function



## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
